cmake/SetupPackages.cmake

```diff
@@ -44,7 +44,8 @@ if (ENABLE_TBB)
   if(TBB_FOUND)
     blt_register_library(
       NAME tbb
-      INCLUDES ${TBB_INCLUDE_DIRS})
+      INCLUDES ${TBB_INCLUDE_DIRS}
+      LIBRARIES tbb)
```

Comment:
My only nitpick is that this should probably be ${TBB_LIBRARIES}, or a condition between the release and debug variants thereof if you're feeling ambitious.